install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2860,7 +2860,7 @@ dnsProxy:
   # -- Maximum number of IPs to maintain per FQDN name for each endpoint.
   endpointMaxIpPerHostname: 50
   # -- Time during which idle but previously active connections with expired DNS lookups are still considered alive.
-  idleConnectionGracePeriod: 0s
+  idleConnectionGracePeriod: 60s
```

Comment:
Can you explain in a bit more depth how you expect this to change the behaviour? For instance, what are your assumptions about FQDN GC frequency vs. CT GC frequency and how this specific configuration flag modifies that behaviour? A time-sequence diagram would go a long way to justify how we think that a 60s buffer here would help.